src/modules/fancyzones/editor/FancyZonesEditor/Models/CanvasLayoutModel.cs

```diff
@@ -179,6 +193,8 @@ protected override void PersistData()
             CanvasLayoutJson jsonObj = new CanvasLayoutJson
             {
                 Uuid = "{" + Guid.ToString().ToUpper() + "}",
+                Eventid = rnd.Next(), // TODO: Give Eventid a unique int

```

Comment:
@MaxBrodeurUrbas correct me if I am wrong, here we used this function as a quick workaround since Guid() doesn't return an int. @markAtMicrosoft  is there a better way of returning a random int that is more optimal than rnd.Next()?